src/styles/blocks/cards__container.scss

```diff
@@ -0,0 +1,29 @@
+.cards__container {
+  padding: 50px 40px;
+  display: grid;
+
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+
+  justify-items: center;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .cards__container{
+    --column-count: 2;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), selectors should be separated by empty lines. Please add an empty line before the `.cards__container` selector inside the media query.